app/registries/foreman/plugin.rb

```diff
@@ -435,6 +435,23 @@ def apipie_ignored_controllers(controllers = nil)
       @apipie_ignored_controllers
     end
 
+    # list of models/render macros paths
+    def apipie_dsl_documented_classes(classes = nil)
+      if classes
+        @apipie_dsl_documented_classes = classes
+        ApipieDSL.configuration.dsl_classes_matchers.concat(classes)
```

Comment:
This is an array: https://github.com/ofedoren/apipie-dsl/blob/c3e7b055a4a80427aa04d20a669b6099bfd14f48/lib/apipie_dsl/configuration.rb#L45

Did you mean `concat!` because this does nothing.

Reply:
Well... As far as I know, `concat` in Ruby adds a new array to `self`. And also, I haven't find `concat!` in the docs :D Unless Rails re-defines this method for arrays..

Reply:
Right, Ruby is consistent, they say... https://ruby-doc.org/core-2.7.0/Array.html#method-i-concat